stylesheets/index.css

```
@@ -0,0 +1,148 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Berkshire+Swash&family=Great+Vibes&display=swap');
+
+* {
+    margin: 0px;
+    padding: 0px;
+    /* box-sizing: border-box; */
+    font-family: 'Poppins', sans-serif;
+}
+
+body {
+    font-family: sans-serif;
+    text-align: center;
+    background-color: #dae1e7;
+}
+
+body {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+}
```

These can be combined into one section:
```suggestion
body {
    font-family: sans-serif;
    text-align: center;
    background-color: #dae1e7;
    display: flex;
    flex-direction: column;
    min-height: 100vh;
}
```

---

Makes sense to combine. Thank you Jasmine!